Error starting the debug session TypeError: is_electron_1.default is not a function

<!-- Please provide a detailed description of the bug. -->
### Bug Description:

The following error occurs when a debug session is started involving a debug adapter contributed by a VSCode plugin:

```
root ERROR Error starting the debug session TypeError: is_electron_1.default is not a function
    at Object.startDebugAdapter (/home/philip/Git/OpenSource/Theia/theia/packages/plugin-ext/lib/plugin/node/debug/plugin-debug-adapter-starter.js:44:48)
    at DebugExtImpl.<anonymous> (/home/philip/Git/OpenSource/Theia/theia/packages/plugin-ext/lib/plugin/node/debug/debug.js:725:82)
    at step (/home/philip/Git/OpenSource/Theia/theia/packages/plugin-ext/lib/plugin/node/debug/debug.js:33:23)
    at Object.next (/home/philip/Git/OpenSource/Theia/theia/packages/plugin-ext/lib/plugin/node/debug/debug.js:14:53)
    at fulfilled (/home/philip/Git/OpenSource/Theia/theia/packages/plugin-ext/lib/plugin/node/debug/debug.js:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

<!-- Please provide clear steps to reproduce the bug. -->
### Steps to Reproduce:

1. Add the VSCode plug-in `webfreak.debug`, e.g. by adding
  `"vscode-native-debug": "https://open-vsx.org/api/webfreak/debug/0.25.1/file/webfreak.debug-0.25.1.vsix"`
  to the root package.json.
2. Run `yarn download:plugins` and `yarn start:browser`
3. Open the Theia browser app in a browser and create a launch config in `.vscode/launch.json`, such as
  ```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Debug",
            "type": "gdb",
            "request": "launch",
            "target": "./hello",
            "cwd": "${workspaceRoot}",
            "valuesFormatting": "parseText",
            "printCalls": true
        }
    ]
}
  ```
4. Go to the debug view in the Theia app and `Start Debugging` the launch config created above
5. The Theia app shows an error notification and prints the error above into the console

<!-- Please provide any additional information available. -->
<!-- Additional information can be in the form of logs, screenshots, screencasts. -->

### Additional Information

- Operating System: Ubuntu 20.04.2
- Theia Version: 1.15.0

Change that introduces the invocation of `isElectron()`:
https://github.com/eclipse-theia/theia/commit/ab27299400b96374643b9139166afcc22195eff3#diff-4cbce14389ca5434c2b22540901b6e30978a267684ca1f6636cb54221559f38b

> If TypeScript allows us we could do `import isElectron = require('is-electron');`.

Unfortunately, TypeScript complains if I do that:
```
packages/plugin-ext/src/plugin/node/debug/plugin-debug-adapter-starter.ts:48:27 - error TS2349: This expression is not callable.
  Type 'typeof import("/home/philip/Git/OpenSource/Theia/theia/node_modules/is-electron/index")' has no call signatures.

48                 execArgv: isElectron()
                             ~~~~~~~~~~


Found 1 error.
```

--- a/packages/plugin-ext/src/plugin/node/debug/plugin-debug-adapter-starter.ts
+++ b/packages/plugin-ext/src/plugin/node/debug/plugin-debug-adapter-starter.ts
@@ -18,7 +18,7 @@ import * as net from 'net';
 import * as theia from '@theia/plugin';
 import { CommunicationProvider } from '@theia/debug/lib/common/debug-model';
 import { ChildProcess, spawn, fork, ForkOptions } from 'child_process';
-import isElectron from 'is-electron';
+const isElectron = require('is-electron');
 
 /**
  * Starts debug adapter process.

